src/transformers/models/auto/modeling_auto.py

```diff
@@ -771,6 +771,13 @@
     ]
 )
 
+MODEL_FOR_ZERO_SHOT_IMAGE_CLASSIFICATION_MAPPING_NAMES = OrderedDict(
```

Comment:
Since we are not using it for an auto class now, maybe mark it as private with an `_` (and adapt the second file)?

Reply:
Done, thanks a lot!